11-Syntax/03-Type-Level-Programming-Syntax/src/02-Basic-Syntax/01-Defining-Custom-Kinds.purs

```diff
@@ -19,11 +19,11 @@ foreign import data Value_Level_Instance2 :: Value_Level_Type_Kind
 -- despite the "foreign import" syntax!
 ----------------------
 
--- Using the value-level type Boolean as an example...
-data Boolean_ = True_Value | False_Value
+-- Using a Boolean-like value-level type as an example...
```

Comment:
I like that you have referred to `Boolean` here as the analogy of `YesNo`. However, it will likely make others wonder why `Boolean` itself wasn't just used, leading them to open a new issue. 

Perhaps this comment should include a link to the corresponding issue?

Reply:
The reason I don't think it would be too problematic is that you already talk about the fact that `Boolean` is an internal type and you change slightly the names in some places so there's no overlap. But sure, once we get the `Boolean` info written it won't hurt to have a link